### PR TITLE
Allow compatibility with Flysystem 3

### PR DESCRIPTION
**Pull request by mihaileu**
#258 this should allow compatibility with the new flysystem 

with the hope that https://github.com/RoyVoetman/flysystem-gitlab-storage/pull/15 will be merged with 3 as major tag.

### CODE REVIEW

**Comment by RoyVoetman** [comment]
https://github.com/RoyVoetman/flysystem-gitlab-storage/releases/tag/v3.0.0

**Comment by mihaileu** [comment]
@bytehead I think we can merge this

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
             - name: Setup PHP
               uses: shivammathur/setup-php@2.7.0
               with:
-                  php-version: 7.4
```
Comment (bytehead) [inline .github/workflows/ci.yml:19]: As we are still compatible with 7.4, why do you remove the tests on 7.4?

**Comment by mihaileu** [comment]
@bytehead  @sheeep !up

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mihaileu) [reply .github/workflows/ci.yml:19]: initially I removed the compatibility with 7.4. I added back, please check if all ok

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -67,7 +67,7 @@ jobs:
...
```
Comment (bytehead) [inline .github/workflows/ci.yml:70]: I would prefer to go on 7.4 as well while checking out the lowest possible dependencies.

**Review by bytehead — CHANGES_REQUESTED** [review]

**Review by bytehead — APPROVED** [review]

**Comment by bytehead** [comment]
Thank you!

**Comment by bytehead** [comment]
[4.4.0](https://github.com/1up-lab/OneupFlysystemBundle/releases/tag/4.4.0) is released - thank you again 🙏

**Pull request merged**